### PR TITLE
nestedInput needed unsetting so it doesn't appear as an attribute

### PR DESCRIPTION
**Pull request by lilHermit**
Subject says it all

### CODE REVIEW

**Comment by ADmad** [comment]
Can you add a test case please.

**Comment by lilHermit** [comment]
Added test method and also fixed issue where input was rendered twice

Non nested
```html
<div class="input text"><label for="foo">Foo</label><input type="text" name="foo" id="foo"/></div>
```

Nested
```html
<div class="input text"><label for="foo"><input type="text" name="foo" id="foo"/>Foo</label><input type="text" name="foo" id="foo"/></div>
```

Fixed Nested
```html
<div class="input text"><label for="foo"><input type="text" name="foo" id="foo"/>Foo</label></div>
```

File: tests/TestCase/View/Helper/FormHelperTest.php [hunk 1]
```diff
@@ -8473,4 +8473,29 @@ public function testAutoDomId()
...
+    /**
+     * testNestedLabelInput method
+     *
+     * Test the `nestedInput` parameter
+     *
+     * @return void
+     */
+    public function testNestedLabelInput()
+    {
+        $result = $this->Form->input('foo', [ 'nestedInput' => true]);
```
Comment (markstory) [inline tests/TestCase/View/Helper/FormHelperTest.php]: Shouldn't be a space after `[`.

**Review by markstory — COMMENTED** [review]
Other than a minor style gripe, this looks good.

File: tests/TestCase/View/Helper/FormHelperTest.php (hunk 1 above)
Comment (lilHermit) [inline tests/TestCase/View/Helper/FormHelperTest.php]: OK I'll correct that tomorrow and push

Comment (markstory) [inline tests/TestCase/View/Helper/FormHelperTest.php]: I can fix it post merge :smile:

**Pull request merged**